includes/class-wcpdf-settings-callbacks.php

```diff
@@ -302,36 +302,36 @@ public function multiple_text_input( $args ) {
 			$fields = isset( $fields_callback_args ) ? call_user_func_array( $fields_callback, $fields_callback_args ) : call_user_func( $fields_callback );
 		}
 
+		printf('<table class="%s multiple-text-input">', esc_attr( $id ) );
 		if ( ! empty( $header ) ) {
-			echo wp_kses_post( "<p><strong>{$header}</strong>:</p>" );
+			echo wp_kses_post( "<tr><td><strong>{$header}</strong>:</td></tr>" );
 		}
-
-		printf('<p class="%s multiple-text-input">', esc_attr( $id ) );
 		foreach ($fields as $name => $field) {
+			echo '<tr>';
 			$size = $field['size'];
 			$placeholder = isset( $field['placeholder'] ) ? $field['placeholder'] : '';
 
-			if ( isset( $field['label_width'] ) ) {
-				$style = sprintf( 'style="display:inline-block; width:%1$s;"', $field['label_width'] );
-			} else {
-				$style = '';
-			}
-
-			$field_description = ! empty( $field['description'] ) ? '<span style="font-style:italic;">'.$field['description'].'</span>' : '';
+			$field_description = ! empty( $field['description'] ) ? $field['description']: '';
 
 			// output field label
 			if ( isset( $field['label'] ) ) {
-				printf( '<label for="%1$s_%2$s" %3$s>%4$s:</label>', esc_attr( $id ), esc_attr( $name ), esc_attr( $style ), esc_html( $field['label'] ) );
+				printf( '<td class="label"><label for="%1$s_%2$s">%3$s:</label></td>', esc_attr( $id ), esc_attr( $name ), esc_html( $field['label'] ) );
```

Comment:
to prevent missing `<td>` elements, this if-statement should now have an "else" too!

Reply:
Fixed here: 5ecad6a